src/styles/_page.scss

```diff
@@ -0,0 +1,78 @@
+//---------------------------------------
+// P
+//---------------------------------------
+.paragraph {
+  @include carbon--type-style('body-long-02');
+  margin-bottom: $spacing-06;
+  padding-right: $spacing-07;
+}
+
+// Responsive by default
+.paragraph--responsive {
+  margin-left: -$spacing-05;
+  padding-left: $spacing-05;
+
+  // 6 col
+  @include carbon--breakpoint('md') {
+    width: 75%;
+  }
+
+  // 8 col
+  @include carbon--breakpoint('lg') {
+    width: 66.667%;
+  }
+}
+
+//---------------------------------------
+// Responsive widths
+//---------------------------------------
+
+.page-h1,
+.page-h2,
+.page-h3,
+.page-h4,
+.page-h5,
+.page-h6,
+.list {
+  width: 100%;
+  @include carbon--breakpoint('md') {
+    width: 75%;
+  }
+  @include carbon--breakpoint('lg') {
+    width: 66.667%;
+  }
+}
+
+//---------------------------------------
+// Headings
+//---------------------------------------
+
+.page-h1 {
+  margin-top: $spacing-12;
+  margin-bottom: $spacing-05;
+  @include carbon--type-style('expressive-heading-05', true);
+}
+
+.page-h2 {
+  margin-top: $spacing-11;
+  margin-bottom: $spacing-05;
+  @include carbon--type-style('expressive-heading-04', true);
+}
+
+.page-h3 {
```

Comment:
@vpicone is there another way to pull in these styles without having to duplicate them??

Reply:
```jsx
import {
  h2,
  h3,
} from 'gatsby-theme-carbon/src/components/markdown/Markdown.module.scss';

import {
  iconCategory,
  categoryTitle
} from './IconLibrary.module.scss';

// then 
 return (
    <section className={iconCategory}>
      <h2 className={cx(h2, categoryTitle)}>{category}</h2>
```

is that what you meant?

Reply:
Yea, that should work I think. thx!